packages/scxa-cell-type-wheel/src/CellTypeWheel.js

```diff
@@ -46,6 +52,17 @@ class CellTypeWheel extends React.Component {
           }
         },
 
+        lang: {
+          noData: `${props.species} doesn't have wheel data for the organism part: ${props.searchTerm}.`
```

Comment:
```suggestion
          noData: `No data exists in our database for this search term: ${props.searchTerm} for ${props.species}.`
```

Reply:
I don't know, I think that was agreed in the team meeting, so @upendrakumbham wrote it down in the issue. https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/455#issue-2478240961